x-pack/docs/en/ml/functions/metric.asciidoc

```diff
@@ -15,6 +15,9 @@ The {xpackml} features include the following metric functions:
 * <<ml-metric-metric,`metric`>>
 * xref:ml-metric-varp[`varp`, `high_varp`, `low_varp`]
 
+NOTE: You cannot add rules with conditions to detectors that use metric 
+functions. 
```

Comment:
I think it should be:

> NOTE: You cannot add rules with conditions to detectors that use the `metric`
> function.

`min`, `max` and `mean` are metric functions, and they conditions should be fine with them.  The reason conditions aren't allowed with `metric` is that `metric` looks at all 3 of `min`, `max` and `mean`, and it would be very hard to reason about which of the 3 statistics the condition had been applied to.